jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/BuildImageStep.java

```diff
@@ -124,14 +124,15 @@
               ? Instant.EPOCH.toString()
               : containerConfiguration.getCreationTime().toString();
       for (int count = 0; count < baseImageLayers.size() - nonEmptyLayerCount; count++) {
-        imageBuilder.addHistory(new HistoryEntry(layerCreationTime, null, null, null));
+        imageBuilder.addHistory(
+            new HistoryEntry(layerCreationTime, null, null, "auto-generated by Jib", null));
```

Comment:
We probably want to go with a builder pattern here since it's unclear which field is which.

Reply:
Also, "auto-generated by Jib" might not be the text we want for `created_by`, since `created_by` refers to what created the layer the history entry refers to (and that layer was not "auto-generated by Jib") - I think "unknown" would be more suitable here.

Reply:
That's `comment`, not `created_by`.

Reply:
Ah right, then my first comment applies haha.